src/main/kotlin/org/rust/ide/presentation/TypeRendering.kt

```diff
@@ -106,6 +106,18 @@ private data class TypeRenderer(
         require(level >= 0)
 
         if (ty == TyUnknown) return unknown
+
+        if (level == 0) return "…"
```

Comment:
@vlad20012 Moving this check up means that some types (primitives) that were previously rendered normally are now rendered as `...` because the level is shifted by one. Is this change OK?

Also, should the aliases be rendered with the same level or one level below?

Reply:
To be honest, I'm not even sure where such "short" rendering is used 😅
It's definitely not used in type hints.
Let's assume for now that this is ok.

Reply:
It only needed a change in [this test](https://github.com/intellij-rust/intellij-rust/pull/7521/files#diff-24bcf1a52e814ac7e5533cc361734126e3b3acea5f81927fcb3f82fd75566b2cR41).